projects/ccf-rui/src/app/shared/components/labeled-side-toggle/labeled-side-toggle.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="organ-sliders">
```

Comment:
This is not what I meant by <ins>**generic component**</ins>! Create a component consisting of a slide-toggle with a label to the right and another label to the left. Then use the <ins>**generic component**</ins> with label inputs `['Left', 'Right']` for one and `['Male', 'Female']` for the other.